docs/settings/settings.md

```diff
@@ -121,6 +121,25 @@ CPU for it from Kubernetes scheduler.
 }
 ```
 
+## `release-download-url`
+
+This setting allows you to configure the upgrade release download URL address. And Harvester will get the ISO URL and checksum value from the `${URL}/${VERSION}/version.yaml` file hosted by the above URL.
+
+Default: `https://releases.rancher.com/harvester`
+
+#### Example
```

Comment:
It makes more sense to tell that it's an example of the version.yaml file not the setting.

```suggestion
#### Example of the version.yaml
```